WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -121,11 +121,10 @@ class DefaultStoresManager: StoresManager {
 
         group.enter()
         synchronizeAccount { _ in
-            group.leave()
-        }
-
-        group.enter()
-        synchronizeAccountSettings { _ in
+            group.enter()
+            self.synchronizeAccountSettings { _ in
```

Comment:
I think it'd be great if you can explicitly state your intention with a capture list - whether to keep `self` weak, strong or unowned. I know that `DefaultStoresManager` is kept alive all through the app's life, so in this case it should not cause trouble with a strong self. But do you think we should keep a weak or unowned self instead?

Reply:
Thanks for pointing this out! I am learning more about memory management now because I keep forgetting to be explicit and consider when to keep `self` weak. I think it makes sense to keep a weak self here, instead. 👍